src/globus_sdk/transport/requests.py

```diff
@@ -114,6 +115,7 @@ def __init__(
         self.verify_ssl = config.get_ssl_verify(verify_ssl)
         self.http_timeout = config.get_http_timeout(http_timeout)
         self._user_agent = self.BASE_USER_AGENT
+        self.globus_clientinfo: GlobusClientInfo = DefaultGlobusClientInfo()
```

Comment:
I would prefer to call this `.globus_client_info` so spell checkers won't argue with us.

Reply:
What if I want to argue with spell checkers?! :stuck_out_tongue_winking_eye: 

That's an easy change for me to get behind.